ui-src/lib/support/TimeFormatter.js

```diff
@@ -116,51 +116,26 @@ export default class TimeFormatter {
     }
     return moment.tz(ms, this._tz).format(format);
   }
-}
 
-/**
- * Specialized class similar to TimeFormatter but forcing a specific time format
- * for internal usage purposes.
- */
-export class TimeStamp90kFormatter {
   /**
-   * Construct from just a timezone specification.
+   * Format timestamp expressed in mill-seconds.
    *
-   * @param  {String} tz Timezone
-   */
-  constructor(tz) {
-    this._formatter = new TimeFormatter(internalTimeFormat, tz);
-  }
-
-  /**
-   * Format a timestamp in 90k units using internal format.
-   *
-   * @param {Number} ts90k timestamp in 90,000ths of a second resolution
+   * @param  {Number} ms     A timestamp in ms to be formatted
    * @return {String}        Formatted timestamp
    */
-  formatTimeStamp90k(ts90k) {
-    return this._formatter.formatTimeStamp90k(ts90k);
+  formatTimeStampMs(ms) {
```

Comment:
Did you mean to add `formatTimeStampMs` and `formatTimeStampSec` back after removing them in 36d5b61?